### ISSUE TITLE
Super brief survey of cheap assets

### ISSUE THREAD

**Issue by ctm** [ctm/mb2-doc#557]
Do an hour or so of web searching (especially including the Unity Assets Store) and see if there are any assets that I should buy as throw-aways to use in a proof-of-concept table.

Yesterday evening, after someone on the AGDG meeting mentioned the "assets store", I found the Unity Assets Store and see that there are some low-cost poker assets. These may be sufficient for me to cobble something together that would be a step up from what we currently have and also allow me to work on the structure of a skinnable table.

This issue is simply to do the survey, but that means I also need to look into what tools I'd use to edit the assets and that's likely to be the time-consuming portion.

**Comment by ctm** [comment]
FWIW, I can probably do a little cheap crowdsourcing by asking the BARGE mailing list in general and perhaps Dan in specific to poke around on my behalf. As such, I should make a post to the BARGE mailing list later today.

**Comment by ctm** [comment]
I just posted to the BARGE mailing list. I didn't get it done yesterday due to ALB #553 and the Team RWB meeting.

**Comment by ctm** [comment]
Well, that was a $15.77 fail. I spent a little time looking around and the Texas Hold'em Poker UI Asset was the only thing possibly worth buying, so at that price, I bought it, although after looking at the license and the contents. Turns out however, once you pay for it, instead of getting something like a tar file or zip file, the only choice is "Open In Unity", which doesn't work for me since I don't have (nor do I want Unity).

So, although I might have been able to hack something together with the various files that were ostensibly contained in what I bought, I'm not going to spend any more time on this issue.
